resources/bin/crafter.sh

```diff
@@ -375,7 +375,7 @@ function elasticsearchStatus(){
 
 function startTomcat() {
   cd $CRAFTER_BIN_DIR
-  if [[ ! -d "$CRAFTER_BIN_DIR/dbms" ]] || [[ -z $(pidOf "$MARIADB_PORT") ]] ;then
+  if [[ ! -d "$CRAFTER_BIN_DIR/dbms" ]] || [[ -z $(pidOf "$MARIADB_PORT") ]] || [[ "$SPRING_PROFILES_ACTIVE" == "crafter.studio.externalDb" ]] ;then
```

Comment:
Don't do an equals comparison since `SPRING_PROFILES_ACTIVE` can have several profile values. Use a regex comparison instead, like in https://github.com/craftercms/craftercms/blob/aa986b43c0b19328095cc4368d87bd6a491bf41f/resources/bin/crafter.sh#L812

Reply:
updated. thanks!